spec/unit/hanami/router/recognize_spec.rb

```diff
@@ -210,6 +227,21 @@
         expect(route.params).to eq({})
       end
 
+      it 'recognizes redirect' do
```

Comment:
@mereghost I don't like it because it's not good english... `expect(route).to redirect_to('/')` would be way better.
Unlike being routable, or truthy, you can't "be redirect": a request can "be redirected" or a route can "be a redirection".